utils/perf.c

```diff
@@ -263,7 +265,7 @@ int setup_perf_data(struct uftrace_data *handle)
 	size_t i;
 	int ret = -1;
 
-	xasprintf(&pattern, "%s/perf-cpu*.dat", handle->dirname);
+	xasprintf(&pattern, "perf-cpu*.dat");
```

Comment:
Why is this change needed?  And I think it changes behavior.  Did you test this?

Reply:
Oh, It seems to make some problem. I'll put it back.

The reason why I changed this line is, the former format `"%s/perf-cpu*.dat"` makes string result like below:
```
reading .//perf-cpu1.dat
2588.701366131   2489: [event] linux:task-name(200006)
2588.704225677   2489: [event] linux:sched-out(200002)
...
```
But as you already pointed it out, I think it can be a matter when user try to print information on the external directory. 
I will trim `globbuf.gl_pathv[i]`. Thank you.